src/components/GroupMgmtPage/index.js

```diff
@@ -6,23 +6,69 @@ import SquareButton from '../SquareButton';
 import RoundedButton from '../RoundedButton';
 import ModalDelete from '../ModalDelete';
 import Banner from '../Banner'
+import Axios from 'axios';
+import APIURL from '../../utils/APIURL';
+import { Button } from 'reactstrap'
 
 class GroupMgmtPage extends React.Component {
     // handleSubmit = submitEvent => {
     // 	submitEvent.preventDefault();
     // 	this.props.handleLogOut();
     // };
 
+    //set initial state of page 
+    state = {
+        existingGroups: []
+    }
+
+    //on page load, axios call to DB, then convert to json, then update state
+    componentDidMount() {
+        Axios.get('http://localhost:8080/api/user/2/group')
```

Comment:
Let's use APIURL here so Heroku works too. I'm also thinking we might want to change the route to pull the user id from the session variable instead. That way no one can use something like Postman to grab any user's groups list. Would '/api/user/groups' be a good route? I can set it up on Bailfire within 20 minutes.

Reply:
Oh right, I forgot to change that. I hardcoded it so I could test. I'm not quite sure how to implement APIURL / route, so maybe when you're in class, we can do it together?